courses/admin.py

```diff
@@ -66,11 +69,31 @@ def get_program(self, obj):
         return obj.program.readable_id if obj.program is not None else None
 
 
+class CourseRunAdminForm(forms.ModelForm):
+    """Admin form for CourseRun"""
+
+    class Meta:
+        model = CourseRun
+        fields = "__all__"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if not self.instance.pk:
+            start_date = now_in_utc()
+            end_date = start_date + timedelta(days=1)
+            self.initial["start_date"] = start_date.replace(
+                hour=23, minute=59, second=0, microsecond=0
+            )
+            self.initial["end_date"] = end_date.replace(
+                hour=23, minute=59, second=0, microsecond=0
+            )
+
```

Comment:
Another option is to override a ModelAdmin method `get_changeform_initial_data` to set the initial dates. It seems to be much simpler. WDYT?

```
    def get_changeform_initial_data(self, request):
        initial = super().get_changeform_initial_data(request)
        start_date = now_in_utc().replace(hour=23, minute=59, second=0, microsecond=0)
        initial["start_date"] = start_date
        initial["end_date"] = start_date + timedelta(days=1)
        return initial
```